packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock.tsx

```diff
@@ -1,150 +1,204 @@
                                             >
-                                                <EuiAccordion
-                                                    id={
-                                                        productBlockInstanceValue.field
-                                                    }
-                                                    arrowDisplay="left"
-                                                    buttonContent={
-                                                        <div>
-                                                            <td
-                                                                valign={'top'}
-                                                                style={{
-                                                                    width: 222,
-                                                                }}
-                                                            >
-                                                                <b>
-                                                                    {
-                                                                        productBlockInstanceValue.field
-                                                                    }
-                                                                </b>
-                                                            </td>
-                                                            <td>
-                                                                {
-                                                                    productBlockInstanceValue.value
-                                                                }
-                                                            </td>
-                                                        </div>
-                                                    }
-                                                ></EuiAccordion>
-                                            </td>
-                                        </tr>
+                                                {ownerSubscriptionId}
+                                            </a>
+                                        )}
+                                    </td>
+                                </tr>
+                                <tr key={-1}>
+                                    <td
+                                        valign={'top'}
+                                        css={productBlockLeftColStyle}
+                                    >
+                                        <b>{t('inUseByRelations')}</b>
+                                    </td>
+                                    <td
+                                        valign={'top'}
+                                        css={productBlockRightColStyle}
+                                    >
+                                        <EuiCodeBlock language="json">
```

Comment:
No it isn't because it is styled for process timeline.

Consider this JSON block temporary; to make it easier to debug the backend for inUseByRelations...